staging.theanvil.io/manifest.json

```diff
@@ -110,6 +173,138 @@
         "path-rewrite": "/",
         "use-tls": "false",
         "ready-probe": "/"
+      },
+      {
+        "target-port": 8888,
+        "cpu-limit": "1.0",
+        "memory-limit": "512Mi",
+        "name": "Jupyter Notebook Bio Python",
+        "image": "quay.io/occ_data/jupyternotebook:1.9.0",
```

Comment:
We should move to `uc-cdis` images from [`containers`](https://github.com/uc-cdis/containers).